pkg/loadbalancer/ipvs.go

```diff
@@ -59,13 +62,28 @@ func NewIPVSLB(address string, port int, forwardingMethod string, backendHealthC
 		log.Errorf("ensure IPVS kernel modules are loaded")
 		log.Fatalf("Error starting IPVS [%v]", err)
 	}
+
 	i, err := c.Info()
 	if err != nil {
 		log.Errorf("ensure IPVS kernel modules are loaded")
 		log.Fatalf("Error getting IPVS version [%v]", err)
 	}
 	log.Infof("IPVS Loadbalancer enabled for %d.%d.%d", i.Version[0], i.Version[1], i.Version[2])
 
+	err = sysctl.WriteProcSys("/proc/sys/net/ipv4/vs/conntrack", "1")
+	if err != nil {
+		log.Errorf("ensure net.ipv4.vs.conntrack is enabled")
+		log.Fatalf("Error ensuring net.ipv4.vs.conntrack enabled [%v]", err)
+	}
+	log.Infof("sysctl set net.ipv4.vs.conntrack to 1")
+
+	err = sysctl.WriteProcSys("/proc/sys/net/ipv4/ip_forward", "1")
+	if err != nil {
+		log.Errorf("ensure net.ipv4.ip_forward is enabled")
+		log.Fatalf("Error ensuring net.ipv4.ip_forward enabled [%v]", err)
+	}
+	log.Infof("sysctl set net.ipv4.ip_forward to 1")
```

Comment:
Hi @thebsdbox @lou-lan please suggest on this part. I can remove this part if both of you think it's not quite making sense.

Reply:
Only masquerade mode will enable IPVS, others won't. So I remove the `if block`. Is anything else I missed?